Lib/idlelib/configdialog.py

```diff
@@ -735,16 +737,16 @@ def create_page_general(self):
         win_height_title = Label(frame_win_size, text='Height')
         self.entry_win_height = Entry(
                 frame_win_size, textvariable=self.win_height, width=3)
```

Comment:
You changed `entry_win_height` and `entry_win_width` in the comment, but not the code.